test/organization/test.sh

```diff
@@ -393,6 +440,109 @@ function test_cdb_usertables_should_work_with_orgusers() {
     sql cdb_testmember_1 "DROP TABLE test_perms_priv"
 }
 
+function test_CDB_Group_Table_GrantRead_should_grant_select_and_RevokeAll_should_remove_it() {
```

Comment:
Going off topic here:

We should start thinking of offloading non organization tests to other suites. I wouldn't like to pollute this file and make it unmanageable.

And start thinking if this is the approach we want to follow for tests. As development in cartodb-postgresql is growing lately we should invest on testing also.

BTW I recently discovered http://pgtap.org/